frontend/src/app.css

```diff
@@ -6,6 +6,9 @@
     Last modified: 12/05/2024
 */
 
+@tailwind base;
```

Comment:
Thanks for adding Tailwind. This file will be where we make common Tailwind components?

Reply:
yep that's right. can see the extracting classes with @apply section of the these docs - https://tailwindcss.com/docs/reusing-styles